spring-cloud-gcp-starters/spring-cloud-gcp-starter-alloydb/.repo.metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "spring-cloud-gcp-starter-alloydb",
```

Comment:
@burkedavison Do we need this? I don't think we provide repo metadata for other modules in this project.

Reply:
All of the starters do have them. (https://github.com/GoogleCloudPlatform/spring-cloud-gcp/tree/main/spring-cloud-gcp-starters)

Whether we need them --> I don't know of a use-case of them today.